app/services/foreman_rh_cloud/cloud_request_forwarder.rb

```diff
@@ -76,6 +82,10 @@ def platform_request?
       ->(request_path) { request_path.include? '/platform' }
     end
 
+    def connection_test_request?
+      ->(request_path) { request_path =~ /redhat_access\/r\/insights\/?$/ }
```

Comment:
If the insights-client  is configured to run with "legacy_upload=False", test connections will come
in via the "/platform" path, so this regex will not catch it.
But I think that is fine because the client calls  "/r/insights/platform/apicast-tests/ping" itself.

Reply:
Double checked, it works with `legacy_upload=False` too